install/linux/usr/share/odemis/sim/sparc2-polarizer-sim.odm.yaml

```diff
@@ -0,0 +1,353 @@
+}
+
+# The first lens of Plate 1, able to move along the whole range
+"Lens1 Mover": {
+    class: actuator.MultiplexActuator,
+    role: "lens-mover",
+    children: {"x": "Optical Actuators"},
+    init: {
+        axes_map: {"x": "l1"},
+        ref_on_init: ["x"],
+    },
+    metadata: {
+        # Default position of the lens (can be improved by user)
+        FAV_POS_ACTIVE: {"x": 0.0045} # m
+    },
+    affects: ["Camera", "Spectrometer 1", "Spectral Camera", "Spectrometer 2"],
+}
+
+# The second lens of Plate 1, either to working or parking position
+"Lens2 Switch": {
+    class: actuator.FixedPositionsActuator,
+    role: "lens-switch",
+    children: {"x": "Optical Actuators"},
+    init: {
+        axis_name: "l2",
+        positions: {
+            4.e-3: "on",
+            0.02: "off", # completely out of the path
+        },
+    },
+    affects: ["Camera", "Spectrometer 1", "Spectral Camera", "Spectrometer 2"],
+}
+
+# Control the slit position to either fully-open or small (dependent on the spectrometer slit-in)
+"Slit": {
+    class: actuator.FixedPositionsActuator,
+    role: "slit-in-big",
+    children: {"x": "Optical Actuators"},
+    init: {
+        axis_name: "slit",
+        positions: {
+            0: "on", # fully opened
+            -0.0012: "off", # opening based on the small slit
+        },
+    },
+    affects: ["Camera", "Spectrometer 1", "Spectral Camera", "Spectrometer 2"],
+}
+
+# Controller for moving the 'Redux' stage
+# Note: the S axis can _only_ be moved when the l axis is near the active/engaged position
+"Mirror Actuators": {
+    class: tmcm.TMCLController,
+    role: "mirror",
+    init: {
+        port: "/dev/fake6",
+        address: null,
+        axes: ["s", "l"],
+        ustepsize: [1.e-6, 1.e-6], # m/µstep (big, for the simulator to move fast)
+        refproc: "Standard",
+        refswitch: {"s": 0, "l": 0},
+    },
+    metadata: {
+        # Default position of the mirror engaged (will be improved during alignment)
+        FAV_POS_ACTIVE: {"l": 70.e-3, "s": 5.e-3},  # m, m
+    },
+}
+
+# Internal component to convert between the referential of the mirror actuators
+# and the X/Y referential of the SEM. Used by the mirror alignment functions.
+"Mirror Actuators in XY referential": {
+    class: actuator.ConvertStage,
+    role: "mirror-xy",
+    children: {"orig": "Mirror Actuators"},
+    init: {
+        axes: ["l", "s"], # mapped respectively to X, Y
+        rotation: -1.0471975511965976, # rad (= 60°)
+        # Note: if you want to use absolute moves, add a "translation" too
+    },
+}
+
+"Polarization Analyzer": {
+    class: actuator.CombinedFixedPositionActuator,
+    role: "pol-analyzer",
+    children: {"Pol1": "Linear Polarizer", "Pol2": "Quarter Wave Plate"}, # alphabetically sorted
+    init: {
+        axis_name: "pol",
+        caxes_map: ["rz", "rz"],
+        atol: [3.392e-5, 3.392e-5], # rad (~the step size of the motor)
+        fallback: "unspecified position",
```

Comment:
We need a better name for the "unspecified position", because 1) it's long  and 2) it contains a space. I'd fine with "unspecified" or "unknown" or maybe even `None` (aka `null` in YAML).